scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -292,10 +292,14 @@
         pod-name (:task-id task-metadata)
         ^V1Pod pod (api/task-metadata->pod pod-namespace compute-cluster task-metadata)
         new-cook-expected-state-dict {:cook-expected-state :cook-expected-state/starting
-                                      :launch-pod {:pod pod}}]
+                                      :launch-pod {:pod pod}}
+        job-map (get-in task-metadata [:task-request :job])
+        job-uuid (str (:job/uuid job-map))]
```

Comment:
Can we destructure `:job/uuid`?

Reply:
So it would be created in the same line as `job-map`? I wasn't quite sure how to do that in Clojure so I just kept it like this. If there's a good way to do it I'd be interested to learn.

Reply:
Yeah, like I think instead of this:

```
job-map (get-in task-metadata [:task-request :job])
job-uuid (str (:job/uuid job-map))]
```

you could have this?:

```
{:keys [job/uuid] :as job-map} (get-in task-metadata [:task-request :job])
job-uuid (str uuid)]
```